com.dropbox.Client.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id": "com.dropbox.Client",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "eos-dropbox-app",
+    "homepage": "https://www.dropbox.com",
+    "cleanup": [
+        "/lib/debug"
+    ],
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Sound and network access */
+        "--socket=pulseaudio",
```

Comment:
What does dropbox need sound for?

Reply:
I have no clue, really, but I decided to keep it in case there was some "beep" sound somewhere I was missing (I don't really use Dropbox myself much, so I did err on the "safe" side here).

I can remove it if you want / feel confident there should not be any sound coming out of this.